test/adapter/json_api/links_test.rb

```diff
@@ -7,18 +7,24 @@ class LinksTest < ActiveSupport::TestCase
         LinkAuthor = Class.new(::Model)
         class LinkAuthorSerializer < ActiveModel::Serializer
           link :self do
-            href "//example.com/link_author/#{object.id}"
+            href "http://example.com/link_author/#{object.id}"
```

Comment:
I did https in the docs but tried to vary it more here to convey what you pass in is what you get back. It used to be `//example.com...` which, admittedly, confused me when I recently discovered that JSON API requires links to be full URLs. They cannot be relative. I have no reservations about making them all https.